modules/maintaining-bare-metal-hosts.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+[id="maintaining-bare-metal-hosts_{context}"]
+= Maintaining bare metal hosts
+
+You can maintain the details of the bare metal hosts in your cluster from the {product-title} web console. Navigate to *Compute* -> *Bare Metal Hosts*, and select a task from the *Actions* drop down menu. Here you can manage items like BMC details, boot MAC address for the host, enable power management, etc. Details of the network interfaces and disks for the host can be reviewed.
+
+You can also move a host into maintenance mode. When you move a bare metal host into maintenance mode, all managed workloads are moved off the corresponding bare metal node. No new workloads are scheduled while in maintenance mode.
+
+You can deprovision a bare metal host. This deletes the machine and scales down the related machine set.
```

Comment:
If this is true it's purely a UI thing. Deprovisioning a BareMetalHost directly does not delete the Machine or scale down the MachineSet (and is also not recommended). The correct way to do this is to annotate the Host you want to deprovision and then scale down the MachineSet (it's possible this is implemented in the UI).

Reply:
@rawagner can you comment on this please. what occurs when you select to deprovision a machine in the UI? Does the system just scale down the machineset?
![image](https://user-images.githubusercontent.com/74046732/119475113-bf4f4080-bd44-11eb-8fdc-e14a87b1a5e0.png)

Reply:
K, so what happens with the deprovision action is: 1) Annotate host’s machine with cluster.k8s.io/delete-machine: true, 2) Scale down the cluster using MachineSet.

Reply:
I've updated with this. @zaneb Can you give these updates the once over again please?